### DISCUSSION TITLE
how use with webpack5、vuepress、vue2

### DISCUSSION THREAD

**Discussion by xu455255849** [vuepress/core#430] [Q&A] [answered]
i use webpack5， vuepress1.x is not work，
Error: Rule can only have one resource source (provided resource and test + include + exclude) in {......

upgrade to @vuepress2 ，but vue2 seem to not support

how can i resolve this

**Comment by Mister-Hope** [comment, marked as answer]
VuePress has a huge deps tree, and it's common that some part of the deps tree is broken. So maybe you should try to create a monorepo and install docs and source code part deps sperately.

**Reply by xu455255849** [reply]
If it can't be solved，Just do that
